Remotion/Web/Core/UI/Controls/WebTreeView.cs

```diff
@@ -174,6 +179,7 @@ public WebTreeView (IControl? ownerControl)
       _renderingFeatures = SafeServiceLocator.Current.GetInstance<IRenderingFeatures>();
       _fallbackNavigationUrlProvider = SafeServiceLocator.Current.GetInstance<IFallbackNavigationUrlProvider>();
       _labelReferenceRenderer = SafeServiceLocator.Current.GetInstance<ILabelReferenceRenderer>();
+      _updateMessageText = string.Empty;
```

Comment:
should be type `WebString?`

Reply:
Not sure, it is only used in Typescript, would that even make sense? We need to convert it to a string anyways?

Reply:
oh, yes, you are correct. I missed this detail in the review. Alas, we now have to make sure the Javascript parameter value is escaped in RenderContents.

Reply:
`ScriptUtiity.EscapeClientScript`